android/build.gradle

```diff
@@ -32,5 +32,8 @@ allprojects {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url "$rootDir/../node_modules/react-native/android"
         }
+        maven {
```

Comment:
Part of bugsnag setup docs

Reply:
We already have this by using `google()`, see https://developer.android.com/studio/build/dependencies#google-maven. Also as you can see there `google()` is relatively new. So I assume the Bugsnag docs have not been updated, yet.